lib/upperkut/batch_execution.rb

```diff
@@ -20,17 +20,23 @@ def execute
       @worker.server_middlewares.invoke(@worker, items) do
         worker_instance.perform(items_body.dup)
       end
-    rescue Exception => ex
-      @worker.push_items(items_body)
-
+    rescue StandardError => error
       @logger.info(
         action: :requeue,
-        ex: ex,
+        ex: error,
```

Comment:
ex?

Reply:
:man_shrugging: 
Isn't that some Logger pattern?